### PR TITLE
Add tiles service

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: templates/tiles-service.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.fullname" ( dict "root" $ "service" $.Values ) }}-tiles
+  labels: {{ include "common.labels" ( dict "root" $ "service" $.Values ) | nindent 4 }}
```
Comment (sbrunner) [inline templates/tiles-service.yaml:5]:
```suggestion
  name: {{ include "common.fullname" ( dict "root" . "service" .Values ) }}-tiles
  labels: {{ include "common.labels" ( dict "root" . "service" .Values ) | nindent 4 }}
```

File: templates/tiles-service.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+spec:
+  type: ExternalName
+  externalName: {{ .Values.mutualize.tccService }}
+  ports:
+  - name: http
+    protocol: TCP
+    port: 80
+    targetPort: 80
+  selector: {{- include "common.selectorLabels" ( dict "root" . "service" .Values ) | nindent 4 }}
```
Comment (sbrunner) [inline templates/tiles-service.yaml:14]: What is this for?

Comment (arnaud-morvan) [reply templates/tiles-service.yaml:14]: This is a copy paste from other services.

**Comment by arnaud-morvan** [comment]
I merge to be able to tests, comments still welcome.

**Pull request merged**